.github/workflows/codeql.yml

```diff
@@ -0,0 +1,37 @@
+name: "CodeQL"
```

Comment:
Docs PR with CodeQL workflow? 😕

Reply:
Yes! This must have been a bad merge that I introduced somehow.